pipelines/broad/dna_seq/germline/single_sample/wgs/WholeGenomeGermlineSingleSample.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 3.1.14 
```

Comment:
We should update the version number in the [WGS Overview doc](https://broadinstitute.github.io/warp/docs/Pipelines/Whole_Genome_Germline_Single_Sample_Pipeline/README).